docs/wallets/lightning.md

```diff
@@ -0,0 +1,13 @@
+# Lightning Network (LN)
+
+---
+
+## What is Lightning (LN)?
+
+The Lightning Network (LN) is a "Layer 2" [protocol](https://en.wikipedia.org/wiki/Lightning_Network) that runs on top of the base layer Decred blockchain ("Layer 1"). It enables fast, low cost transactions and higher throughput than possible on Layer 1. 
```

Comment:
LN is a Layer 2 Protocol, if the paragraph refers generaly LN, you need to say: `that runs on top of blockchain-based cryptocurrency (like Decred)`

Reply:
Good suggestion. Went with:

>The Lightning Network (LN) is a "Layer 2" [protocol](https://en.wikipedia.org/wiki/Lightning_Network) that runs on top of the base layer ("Layer 1") of a blockchain-based cryptocurrency (like Decred).